tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -121,7 +177,9 @@ def prepare_range_checksum_data():
 
 @pytest.fixture(scope="function")
 def pre_insert_agents():
-    """Insert agents. Only used for the global queries"""
+    '''
+    Insert agents. Only used for the global queries.
+    '''
```

Comment:
I would not change this line because it follows the Google style correctly, with a one-line summary. Also, try to use `"""` instead.
```suggestion
   """Insert agents. Only used for the global queries."""
```

Reply:
Done in the following [commit](https://github.com/wazuh/wazuh-qa/commit/5ceb7bd05d809dc92ccde0df5d314a5914036df8)